components/OrganizationEdit/OrganizationEdit.js

```diff
@@ -318,7 +324,7 @@ class OrganizationEdit extends Component {
           </button>
         </div>
         <div className={s.phonesBox}>
-          {organization.phones.map((phone, index) => (
+          {organization.phones && organization.phones.map((phone, index) => (
```

Comment:
This seem redundant given the check above. Can you consolidate to just a single place?